cmd/vcert/validators.go

```diff
@@ -691,3 +693,86 @@ func validateSetPolicyFlags(commandName string) error {
 
 	return nil
 }
+
+func validateSshEnrollFlags(commandName string) error {
+
+	err := validateConnectionFlags(commandName)
+	if err != nil {
+		return err
+	}
+
+	if flags.sshCertKeyId == "" {
+		return fmt.Errorf("a ssh certificate key id is requiered")
+	}
+
+	err = validateExistingFile(flags.sshCertKeyId)
+	if err != nil {
+		return err
+	}
+
+	if flags.sshCertTemplate == "" {
+		return fmt.Errorf("ca value is required")
```

Comment:
"CA value is required (--ca)"

Reply:
as we changed the value to be template then I wrote something like this: "certificate issuing template value is required (--template)" please let me know if you agree with this or you have some changes on that.